ladybug/debugger/src/main/resources/springIbisTestTool.xml

```diff
@@ -76,6 +76,8 @@
 		<property name="threadsTime" value="-1"/>
 	</bean>
 
+	<bean name="customReportAction" class="org.frankframework.ladybug.larva.ConvertToLarvaAction"/>
+
```

Comment:
```suggestion
```

Reply:
This change should be implemented together with the @Component suggested on the public class ConvertToLarvaAction (should have the same effect)